examples/notebooks/Read-plot-Sigmet-data-from-AWS.ipynb

```diff
@@ -0,0 +1,409 @@
+{
```

Comment:
<p>Can you add a link to the bucket metadata on AWS?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openradar/xradar/pull/106/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Added the bucket metadata link on AWS. In the first version it was there. Not sure when it got lost</p><p> </p>